app/assets/javascripts/finder.js

```diff
@@ -0,0 +1,154 @@
+/* global instantsearch */
+
+//CHANGE ME!
+app({
+    appId: 'YIQONPL50S',
+    apiKey: 'bdb6aa5a3997de1fa9aecbade22f7e44',
```

Comment:
should this be secret?
Remember that this is an open source repository, everybody sees this codes..
If it's secret it should be loaded via config/secrets.rb and config/application.yml